public/locales/de/editor.json

```
@@ -0,0 +1,16 @@
+{
+  "editor-desktop-optimized": "Diese Seite ist für Desktop optimiert",
+  "editor-visit-desktop": "Bitte besuche readme.so auf einem Desktop um deine Readme zu erstellen!",
+  "nav-download": "Herunterladen",
+  "download-readme-generated": "Readme generiert!",
```

```suggestion
  "download-readme-generated": "Readme erstellt!",
```

I think `Readme erstellt!` sounds more natural than `Readme generiert!`.

---

Sounds good for me as Swiss German native too.